ssl_cert_tracker/new_tasks.py

```diff
@@ -27,109 +31,117 @@ def go_node(node_id, node_address):
         country_name = ''
         not_before = ''
         not_after = ''
-        ip_value = ''
         md5 = ''
         sha1 = ''
         host_name = ''
         bits = ''
         name = ''
         sig_algo = ''
-    
+        record = {}
+       
```

Comment:
the --options argument for the nmap process call must not be hard coded. you are still not separating nmap calls by port